src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -148,6 +148,11 @@ def build_parser():
                         help = "Skip preprocessing and start running from variant calling. Implies --skip_alignment.",
                         action = 'store_true')
 
+    # what directory are our sequence files in?
+    parser.add_argument('-SD', '--sequence_dir',
```

Comment:
Yeah, I think we're at the point where the short arguments aren't very useful. However, we've got a lot of them in here. Would it be OK with you to hold off on changing this for now, and to then go back after we're done and refactor them all? If so, I'll open a ticket to track.